viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -718,6 +780,9 @@ extern int viam_carto_lib_init(viam_carto_lib **ppVCL, int minloglevel,
     if (vcl == nullptr) {
         return VIAM_CARTO_OUT_OF_MEMORY;
     }
+    if (google::IsGoogleLoggingInitialized()) {
```

Comment:
remove as we can't depend on this existing on linux as linux's version of glog is much older than the one installed by brew on osx.